src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,28 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeId, onTabSelected }) => {
+  const activeTab = tabs.find(tab => tab.id === activeId) || tabs[0];
+
+  return (
+    <div>
+      <ul>
+        {tabs.map(tab => (
+          <li data-cy="Tab" className={tab.id === activeId ? 'is-active' : ''}>
+            <a
+              href={`#${tab.id}`}
+              data-cy="TabLink"
+              onClick={() => {
+                if (tab.id !== activeId) {
+                  onTabSelected(tab.id);
+                }
+              }}
+            >
+              {tab.title}
+            </a>
+          </li>
```

Comment:
Each `<li>` element rendered inside the `tabs.map()` function should have a `key` prop. This is important for React to efficiently update and manage the list of tabs. You can use `tab.id` as the key since it is unique for each tab.